compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lower_to_ukernel.mlir

```diff
@@ -295,3 +295,48 @@ func.func @zero_fill_matmul_elmwise(%arg0 : tensor<8x16x4x8xbf16>, %arg1 : tenso
 // CHECK: linalg.generic
 // CHECK-SAME: iterator_types = ["parallel", "parallel", "parallel", "parallel"]
 // CHECK: return
+
+// -----
+
+#executable_target_amdaie_xclbin_fb = #hal.executable.target<"amd-aie", "amdaie-xclbin-fb", {target_arch = "chip-tbd", ukernels = "all"}>
```

Comment:
This is testing that the ukernel name/call being generated has identified the intrinsics correctly and because of this a rank-2 matmul wouldn't work. The rank needs to be either 4 or 6 (the latter for pack-peel pipeline that we want to focus and ensure the intrinsics for).

Reply:
Is it strix specific? Is it testing functionality added in this PR?

Reply:
Yes, this is Strix specific and basically adds tests in other passes of the pipeline which would transitively ingest the output by the main functionality (`KernelDispatch` related) that's added in this PR.

I've even done a similar check for the e2e lit test.